{{cookiecutter.project_name}}/entrypoint.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -e
```

Comment:
We should not have the Docker entrypoint; it's part of our internal build process